content/en/monitors/notify/variables.md

```diff
@@ -303,6 +202,163 @@ This is the escalation message @dev-team@company.com
+Log monitors, Trace Analytics monitors, RUM monitors and Event monitors can use facets as variables if the monitor is grouped by facet. If a log monitor is grouped by `@facet_key`, use the variable:
+
+```text
+{{ @facet_key.name }}
+```
+
+**Example**: To include group-specific information in a multi alert log monitor group by `@machine_id`:
+
+```text
+This alert was triggered on {{ @machine_id.name }}
+```
+
+If your facet has periods, use brackets around the facet, for example:
+
+```text
+{{ [@network.client.ip].name }}
+```
+
+#### Matching attribute/tag variables
+
+_Available for [Log monitor][2], [Trace Analytics monitor][3] (APM) and [RUM monitor][4]_
+
+To include **any** attribute or tag from a log, a span or a RUM event matching the monitor query, use the following variables :
+
+| Monitor type    | Variable syntax                                         |
+|-----------------|---------------------------------------------------------|
+| RUM             |  `{{rum.attributes.key}}` / `{{rum.tags.key}}`          |
+| Trace Analytics |  `{{span.attributes.key}}` / `{{span.tags.key}}`        |
+| Log             |  `{{log.attributes.key}}` / `{{log.tags.key}}`          |
+
+For any `key:value` pair, the variable `{{log.tags.key}}` renders `value` in the alert message.
+
+**Example**: if a log monitor is grouped by `@error.code`, to include the error message in the notification message, use the variable :
+
+```text
+{{ log.attributes.[error.message] }}
+```
+
+The message renders the `error.message` attribute of a chosen log matching the query, **if the attribute exists**.
+
+<div class="alert alert-info"><strong>Note</strong>: If the picked event does not contain the attribute or the tag key, the variable renders empty in the notification message. To avoid missing notifications, using these variables for routing notification with {{#is_match}} handles is not recommended.</div>
+
+##### First level attributes
+
+Logs, spans and RUM events have generic first level attributes, that you can also use in variables following the following syntax :
+
+| Monitor type    | Variable syntax                       | First level attributes |
+|-----------------|---------------------------------------|------------------------|
+| RUM             | `{{rum.key}}`                         | `service`, `status`, `timestamp` |
+| Trace Analytics | `{{span.key}}`                        | `env`, `operation_name`, `resource_name`, `service`, `status`, `span_id`, `timestamp`, `trace_id`, `type` |
+| Log             | `{{log.key}}`                         | `message`, `service`, `status`, `source`, `span_id`, `timestamp`, `trace_id` |
+
+If the matching event does not contain the attribute in its definition, the variable is rendered empty.
+
+##### Explorer link
+
+Use `{{log.link}}`, `{{rum.link}}` and `{{span.link}}` to enrich the notification with a link to the log, rum or trace explorer, scoped on the events matching the query.
+
+#### Check monitor variables
```

Comment:
Sounds like we should use Tabs instead of sections to put them on the same level.